docs/docs/40-contributor-guide/10-hacking-on-kargo.md

```diff
@@ -18,6 +18,13 @@ execute tests within a container that is maximally similar to those used in CI.
 
 To take advantage of this, you only need to have
 [Docker](https://docs.docker.com/engine/install/) and `make` installed.
+> you can use podman by creating an environment variable
+
+```bash
+export CONTAINER_RUNTIME=podman
+```
+
+<!-- [How to config podman socket](https://docs.podman.ioen/latest/markdown/podman-system-service.1.html#examples) -->
```

Comment:
Can we remove these podman-specific instructions and links to the external site? I'd like to keep docs to a minimum and devs will be able to figure this out themselves.

Reply:
to remove it completely!? how do people know they have the option? why Docker is used as a sample and with links to how and where to install it? what is the diff?

Reply:
For better or worse, Docker is the de facto standard in this space.

The changes to the Makefile are welcome because they add a little flexibility for someone such as yourself who really wants to use an alternative to Docker, and the changes don't complicate things disproportionately to that new flexibility.

Where the docs are concerned, however, to be very candid, it's  not important to us, or even desirable, that they be modified in a way that promotes Red Hat's preference in this space.

For the same reason, you could expect that we're not going to change docs to promote OpenShift Local.

Reply:
Ok, so where i can add a Hint of this possibility?

Reply:
We're asking you not to.

Reply:
Got it, removed everything